quantum_chess/__init__.py

```diff
@@ -0,0 +1,3 @@
+# Quantum Chess no longer exists in this repo. See README.md for mre info.
+
+raise ImportError
```

Comment:
you need to `raise ImportError()` so it actually constructs the exception object.

You can also add a helpful message: `raise ImportError("quantum_chess moved to github.com/quantumlib/unitary/")`